editor/store/reducer.js

```diff
@@ -1001,13 +1001,50 @@ export const sharedBlocks = combineReducers( {
 	},
 } );
 
+/**
+ * Reducer that for each block uid stores an object that represents its nested nestings.
```

Comment:
Do we need to track each block here? Or is it just the context of the currently selected block?

Reply:
What is a "nested nesting" ?

Reply:
Hi @aduth, "nested nesting" was a typo it was supposed to be nested settings.

> Do we need to track each block here? Or is it just the context of the currently selected block?

This is complex, right now, in our use cases only the inserter is using the information for the selected block. But this information should be available for all the blocks and not just the selected one. We may add new settings that affect not just the block that is currently selected.
Another important case is drag&drop, we will need to get the allowed blocks not just for the selected block but for the block we are dragging into so we can check if the dragged block can be nested inside it.